pkg/inject/sidecar.go

```diff
@@ -18,7 +18,8 @@ import (
 var (
 	// Annotation is the annotation name to look for when deciding whether or not to inject
 	Annotation = "sidecar.jaegertracing.io/inject"
-
+	// Label is the label for query to all sidecars of an specific instance.
+	Label = "sidecar.jaegertracing.io/inject"
```

Comment:
Just to confirm, are you adding a label bc you cannot query base on annotations? 

I think there is no need to define a separate `Label string` you can reuse annotation

Reply:
Yes, that is the reason, well ok I'll use the annotation constant, I did in this way just for readability, and well.. I think it doesn't hurt.

Reply:
Did the change, removed Label constant and used Annotation instead.